tests/app/user/test_rest.py

```diff
@@ -72,6 +72,7 @@ def test_get_user(admin_request, sample_service, sample_organisation):
     assert fetched['permissions'].keys() == {str(sample_service.id)}
     assert fetched['services'] == [str(sample_service.id)]
     assert fetched['organisations'] == [str(sample_organisation.id)]
+    assert not fetched['can_use_webauthn']
```

Comment:
Minor: You could consider changing this assert to `is False`, as that way it excludes a variety of things (empty lists/dicts/strings and None) that we wouldn't expect on the front end.

```suggestion
    assert fetched['can_use_webauthn'] is False
```

Reply:
Ahh, I previously tried `== False`, but Flake didn't like that (and suggested "if not cond").

Agree it's better to be explicit 👍.

Reply:
yeah, frustratingly pep8 and various guidance on blogs/stackoverflow is always tilted towards `if` statements in code, where good variable naming and sensible return values can help deal with that, rather than asserts, where we should be as explicit as possible